### ISSUE TITLE
How to access a cookie on both platforms

### ISSUE THREAD

**Issue by juanpmarin** [rangle/angular-ssr#84]
Hi, I need to access a cookie from server and browser, I thought about use a Varian but the cookie has not static values (it's an auth token), do you know a way to access it ? 
Thanks !

**Comment by triadiprabowo** [comment]
@juanpmarin have you find this solution?

**Comment by juanpmarin** [comment]
@triadiprabowo yes, but not using angular-ssr, I had to migrate to angular 5 and the express universal engine, angular 5 now comes with state transfer out of the box

**Comment by juanpmarin** [comment]
@triadiprabowo this is the implementation that I based on
https://github.com/patrickmichalina/fusebox-angular-universal-starter/blob/master/src/client/app/shared/services/cookie.service.ts

**Comment by triadiprabowo** [comment]
@juanpmarin thank you man, it is working now. I used universal-starter (angular 5) and some setup from your given repo for cookies and state transfer.
